src/routes/Competition.jsx

```diff
@@ -17,33 +17,33 @@ const Competition = () => {
       <Container fluid>
         <Row className="my-4 mx-0 gx-5 justify-content-left">
           <Col className="" lg={{ span: 6 }}>
-            <Content title="INTERCOLLEGIATE ROCKET ENGINEERING COMPETITION">
+            <Content title="LAUNCH CANADA">
               <p>
-                The IREC is an international competition each June in Las Cruces,
-                New Mexico. Hosted by the Experimental Sounding Rocket Association
-                in partnership with Spaceport America, it is the flagship event
-                of the Spaceport America Cup. Student teams compete in one of six
-                categories, split between two different altitude categories and
-                three different engine types. Each rocket carries a four kilogram
-                payload. Students are strongly encouraged to design and construct
-                each of their rocket&apos;s myriad systems themselves: everything from
-                propulsion and structures to avionics, recovery, and payload.
-                Overuse of commercial components is discouraged. In this way, the
-                IREC promotes student design and a spirit of simple, holistic
-                design: excess complexity is the bane of the student rocket
-                engineering team.
+                The Launch Canada Challenge is Canada’s
+                first-ever major national rocketry competition.
+                It brings together Canada’s passionate and
+                rapidly growing amateur and student rocketry community
+                with academia, industry and government to facilitate
+                the development of national talent in the interdisciplinary
+                field of rocket engineering, while fostering a strong
+                culture of safety.
+              </p>
+              <p>
+                The team have competed in the 2023 Launch Canada challenge
```

Comment:
nit: since the team is acting as a whole here, it should be "the team has competed"